From a57b3e2be4e6ac6b11e12f440fe6781ae9ce5c39 Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Fri, 25 Sep 2009 09:39:44 +0200
Subject: [PATCH 1/2] rygel.conf: enabled by default.

---
 data/rygel-default.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/rygel-default.conf b/data/rygel-default.conf
index 55cc03e..3a66085 100644
--- a/data/rygel-default.conf
+++ b/data/rygel-default.conf
@@ -6,7 +6,7 @@
 # General configuration options
 [general]
 # Set it to 'true' if Rygel should be run as part of user's session.
-enabled=false
+enabled=true
 
 # Set it to 'false' if you want to disable transcoding support.
 enable-transcoding=true
-- 
1.6.4.3

