From 23969b86949b15a536f3757d569fe73d3beb1a3a Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Fri, 25 Sep 2009 09:39:44 +0200
Subject: [PATCH 1/2] rygel.conf: enabled by default.

---
 data/rygel-default.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/rygel-default.conf b/data/rygel-default.conf
index df71da3..7c49686 100644
--- a/data/rygel-default.conf
+++ b/data/rygel-default.conf
@@ -6,7 +6,7 @@
 # General configuration options
 [general]
 # Set it to 'true' if Rygel should be run as part of user's session.
-enabled=false
+enabled=true
 
 # Set it to 'false' if you want to disable transcoding support.
 enable-transcoding=true
-- 
1.7.0

