From c79a3dc727f86a2097175d982e1f7bef64fdc92a Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Tue, 19 Jan 2010 10:25:00 +0000
Subject: [PATCH 2/2] enable rygel by default in maemo config as well.

---
 data/rygel-maemo.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/rygel-maemo.conf b/data/rygel-maemo.conf
index e4417fb..f2e74f6 100644
--- a/data/rygel-maemo.conf
+++ b/data/rygel-maemo.conf
@@ -6,7 +6,7 @@
 # General configuration options
 [general]
 # Set it to 'true' if Rygel should be run as part of user's session.
-enabled=false
+enabled=true
 
 # Set it to 'false' if you want to disable transcoding support.
 enable-transcoding=true
-- 
1.7.0

