diff --git a/configure b/configure
index 44c290f..51cb3d3 100755
--- a/configure
+++ b/configure
@@ -11636,7 +11636,7 @@ GIO_REQUIRED=2.16
 GEE_REQUIRED=0.5.0
 UUID_REQUIRED=1.41.3
 LIBSOUP_REQUIRED=2.26.0
-GTK_REQUIRED=2.90.3
+GTK_REQUIRED=2.16
 LIBSQLITE3_REQUIRED=3.5
 
 
@@ -12919,12 +12919,12 @@ if test -n "$GTK_CFLAGS"; then
     pkg_cv_GTK_CFLAGS="$GTK_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gtk+-3.0 >= \$GTK_REQUIRED \""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gtk+-3.0 >= $GTK_REQUIRED ") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gtk+-2.0 >= \$GTK_REQUIRED \""; } >&5
+  ($PKG_CONFIG --exists --print-errors "gtk+-2.0 >= $GTK_REQUIRED ") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_GTK_CFLAGS=`$PKG_CONFIG --cflags "gtk+-3.0 >= $GTK_REQUIRED " 2>/dev/null`
+  pkg_cv_GTK_CFLAGS=`$PKG_CONFIG --cflags "gtk+-2.0 >= $GTK_REQUIRED " 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -12935,12 +12935,12 @@ if test -n "$GTK_LIBS"; then
     pkg_cv_GTK_LIBS="$GTK_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gtk+-3.0 >= \$GTK_REQUIRED \""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gtk+-3.0 >= $GTK_REQUIRED ") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gtk+-2.0 >= \$GTK_REQUIRED \""; } >&5
+  ($PKG_CONFIG --exists --print-errors "gtk+-2.0 >= $GTK_REQUIRED ") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_GTK_LIBS=`$PKG_CONFIG --libs "gtk+-3.0 >= $GTK_REQUIRED " 2>/dev/null`
+  pkg_cv_GTK_LIBS=`$PKG_CONFIG --libs "gtk+-2.0 >= $GTK_REQUIRED " 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -12958,9 +12958,9 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        GTK_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "gtk+-3.0 >= $GTK_REQUIRED " 2>&1`
+	        GTK_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "gtk+-2.0 >= $GTK_REQUIRED " 2>&1`
         else
-	        GTK_PKG_ERRORS=`$PKG_CONFIG --print-errors "gtk+-3.0 >= $GTK_REQUIRED " 2>&1`
+	        GTK_PKG_ERRORS=`$PKG_CONFIG --print-errors "gtk+-2.0 >= $GTK_REQUIRED " 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$GTK_PKG_ERRORS" >&5
@@ -12991,15 +12991,15 @@ $as_echo "yes" >&6; }
 
 
         if test x$enable_vala = xyes ; then
-                as_ac_File=`$as_echo "ac_cv_file_"${VAPIDIR}/gtk+-3.0.vapi"" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for \"${VAPIDIR}/gtk+-3.0.vapi\"" >&5
-$as_echo_n "checking for \"${VAPIDIR}/gtk+-3.0.vapi\"... " >&6; }
+                as_ac_File=`$as_echo "ac_cv_file_"${VAPIDIR}/gtk+-2.0.vapi"" | $as_tr_sh`
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for \"${VAPIDIR}/gtk+-2.0.vapi\"" >&5
+$as_echo_n "checking for \"${VAPIDIR}/gtk+-2.0.vapi\"... " >&6; }
 if { as_var=$as_ac_File; eval "test \"\${$as_var+set}\" = set"; }; then :
   $as_echo_n "(cached) " >&6
 else
   test "$cross_compiling" = yes &&
   as_fn_error "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r ""${VAPIDIR}/gtk+-3.0.vapi""; then
+if test -r ""${VAPIDIR}/gtk+-2.0.vapi""; then
   eval "$as_ac_File=yes"
 else
   eval "$as_ac_File=no"
@@ -13016,7 +13016,7 @@ eval as_val=\$$as_ac_File
 else
 
                         HAVE_GTK=no
-                        as_fn_error "\"Unable to find Vala bindings for gtk+-3.0\"" "$LINENO" 5
+                        as_fn_error "\"Unable to find Vala bindings for gtk+-2.0\"" "$LINENO" 5
 
 fi
 
